client/src/modules/cohort/index.jsx

```diff
@@ -1,11 +1,27 @@
-import React from 'react';
+import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
 
-const Cohort = ({
-  match: {
-    params: { id }
+import Toolbar from 'common/components/Toolbar/Toolbar';
+
+class Cohort extends PureComponent {
+  render() {
+    const {
+      match: {
+        params: { id }
+      }
+    } = this.props;
+
+    return (
+      <Fragment>
+        <Toolbar />
+        <div>
+          Cohort of id:
+          {id}
```

Comment:
why in new line?

Reply:
I know it's strange but I get eslint error: `{id}` must be placed on a new line eslint(react/jsx-one-expression-per-line)